bundle/jsky.app.ot/src/main/resources/resources/conf/welcome.html

```diff
@@ -30,11 +30,7 @@ <h1>Welcome to Observing Tool</h1>
 
 <p>The Observing Tool is the work of the Gemini Observatory staff including Bryan Miller, Andrew Stephens, Mischa Schirmer, and the software engineers in the Gemini Engineering Operations Department and former employees Florian Nussberger, Devin Dawson, Larry O'Brien, Nicolas Barriga, Alan Brighton, Vasudeva Upadhya, Lorraine Callahan, Darrell Denlinger, Kim Gillies, and Phil Puxley.</p>
 
-<p>The image at left shows the interacting galaxy pair NGC 1531 and
-  NGC 1532 imaged with the Gemini Multi-Object Spectrograph (GMOS) on
-  the Gemini South telescope. The larger NGC 1532 is slightly more
-  massive than the Milky Way while the smaller NGC 1531 is comparable
-  to the Large Magellanic Cloud. Credit: Gemini Observatory.</p>
+<p>The image at the left shows an M-band (4.68 micron) mosaic of Jupiter taken with NIRI on the Gemini North telescope on January 11, 2017; the Great Red Spot is visible just below center. This image was made by combining the sharpest short exposure images; an approach commonly called “lucky imaging.” At these infrared wavelengths, Jupiter glows with thermal emission, and the dark areas indicate where thick clouds are blocking the emission from the deeper atmosphere. These data were obtained in support of the Juno mission, taken close in time to the spacecraft's passages by Jupiter. They provide a high-resolution spatial context for Juno’s Jupiter Infrared Auroral Mapper (JIRAM) spectroscopic observations and cover wavelengths and regions of the planet not observed by JIRAM. They also place upper-atmospheric constraints on Jupiter’s circulation in the deep atmosphere determined by Juno’s Microwave Radiometer (MWR) experiment. Image credit: Michael Wong (University of California, Berkeley).</p>
```

Comment:
i guess it would be good to have this as a multiline, but don't bother about fixing it